product_docs/docs/pgd/5/rel_notes/pgd_5.5.1_rel_notes.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "EDB Postgres Distributed 5.5.1 release notes"
+navTitle: "Version 5.5.1"
+---
+
+Released: 31 Mar 2024
+
+EDB Postgres Distributed version 5.5.1 is a patch release containing bug-fixes for EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.5.1. See [PGD/TPA upgrades](../upgrades/tpa_overview) for details.
+!!!
+
+
+## Bug fixes
+
+| Component | Version | Description                                                                                                                                                                                                                                                                                                                                                                   | Ticket |
+|-----------|---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------|
+| BDR       | 5.5.1   | <details><summary>Fix a data inconsistency issue with mixed version usage during upgrades</summary>Upgrading from any previous PGD version to PGD 5.5.0 may result in loss of data when replicating from the newer PGD 5.5.0 node to an older version PGD node, thus leading to inconsistencies between nodes. This release fixes mixed version operation to allow for smooth rolling upgrades.</details> |        |
```

Comment:
@djw-m can we adjust to say "Upgrading from a previous PGD version to PGD 5.5.0 may result in data inconsistencies when replicating from the newer PGD 5.5.0 node to an older version PGD node. This release fixes mixed version operation to allow for smooth rolling upgrades." to soften the language?

Reply:
```suggestion
| BDR       | 5.5.1   | <details><summary>Fix a data inconsistency issue with mixed version usage during upgrades</summary>Upgrading from any previous PGD version to PGD 5.5.0 may result in inconsistencies  when replicating from the newer PGD 5.5.0 node to an older version PGD node. This release fixes mixed version operation to allow for smooth rolling upgrades.</details> |        |
```